.evergreen/config.in.yml

```diff
@@ -865,15 +878,12 @@ functions:
           cp $PROJECT_DIRECTORY/.evergreen/run-mongodb-aws-ecs-test.sh $ECS_SRC_DIR/.evergreen
           tar -czf $ECS_SRC_DIR/src.tgz -C $PROJECT_DIRECTORY .
 
-          cd $AUTH_AWS_DIR
-          cat <<EOF > setup.js
-            const mongo_binaries = "$MONGODB_BINARIES";
-            const project_dir = "$ECS_SRC_DIR";
-          EOF
-
-          cat setup.js
+          set -ex
+          cd ${DRIVERS_TOOLS}/.evergreen/auth_aws
           . ./activate-authawsvenv.sh
-          ${MONGODB_BINARIES}/mongo --nodb setup.js aws_e2e_ecs.js
+          export MONGODB_BINARIES="${MONGODB_BINARIES}";
+          export PROJECT_DIRECTORY=$PROJECT_DIRECTORY;
+          python aws_tester.py ecs
```

Comment:
This will fail our first AWS test that checks that no auth credentials in the URI will fail. It actually connects successfully because the mongod is started without enforcing access control.